Client/Frontend/Browser/Browser.swift

```diff
@@ -356,6 +360,27 @@ class Browser: NSObject {
             self.screenshotUUID = NSUUID()
         }
     }
+
+    @available(iOS 9, *)
+    func toggleDesktopSite(on: Bool) {
+        guard let webView = webView, let currentItem = webView.backForwardList.currentItem else {
+            return
+        }
+
+        if on {
+            let defaults = NSUserDefaults(suiteName: AppInfo.sharedContainerIdentifier())!
```

Comment:
It doesn't feel right to have to create an `NSUserDefaults` instance here. Passing the `NSUserDefaults` instance to `UserAgent` actually seems like overkill to me; we could have `UserAgent` create and hold its own `NSUserDefaults` that we don't need to pass into these methods.

Would you mind moving this into `UserAgent`, then removing the `suiteName` parameter from all the methods currently accepting it?